deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts

```diff
@@ -144,7 +144,7 @@ export class MigrationServiceCore extends Stack {
             cluster: ecsCluster,
             taskDefinition: serviceTaskDef,
             assignPublicIp: true,
-            desiredCount: props.taskInstanceCount ? props.taskInstanceCount : 1,
+            desiredCount: props.taskInstanceCount,
```

Comment:
Will we be automatically starting the metrics collection container?

Reply:
Yes that is the default. Is there a problem there if the Replayer isn't active?

Reply:
sorry - I read this backward for some reason.